GRAPHICS/SetCheckpointIconHeight.md

```diff
@@ -0,0 +1,22 @@
+---
+ns: GRAPHICS
+aliases: ["0x4B5B4DA5D79F1943", "_SET_CHECKPOINT_SCALE"]
+---
+## _SET_CHECKPOINT_ICON_HEIGHT
+
+```c
+// 0x4B5B4DA5D79F1943 0x80151CCF
+void _SET_CHECKPOINT_ICON_HEIGHT(int checkpoint, float height_multiplier);
+```
+
+This multiplies the height of the icon inside a checkpoint with the default height of about 2 units above the checkpoint's coordinates.
+
+## Parameters
+* **checkpoint**: The checkpoint the multiplier should be applied to.
+* **height_multiplier**: The multiplier for the icon height above the checkpoint's coordinates. Default is 1.0
+
+## Examples
+```lua
+local checkpoint = CreateCheckpoint(...)
```

Comment:
This example won't work when copy/pasting, but it's not a major issue, try to make a copy/pastable example in the future.

Reply:
Alright, wasn't aware that they should be copy/pastable without modification. Will keep that in mind!